engine-protocol/src/main/proto/engine_control.proto

```diff
@@ -58,7 +70,10 @@ service EngineControl {
    */
   rpc WaitForIdleState (WaitForIdleStateRequest) returns (WaitForIdleStateResponse);
 
-
+  /*
+    Get all records from the in-memory engine. These records will be returned
+    in a JSON format. Client-side these should be mapped to Records.
+   */
```

Comment:
❓ Should this documentation refer to the specific JSON-schema used for these records? I'm not sure, what do you think?

Reply:
Do we have these schemas available somewhere already? I think only a very select few users would ever implement the gRPC client-side themselves, usually it's handled by the extension. But it can't hurt to add it I suppose.

Reply:
I'm not really sure what the schema is here. But, I guess it refers to https://github.com/camunda-cloud/zeebe/tree/main/protocol. If it's about deserialization, we can refer to https://github.com/camunda-cloud/zeebe/tree/main/protocol-jackson as an easy way to achieve the mapping.